components/httpd_server/src/httpd/session.hpp

```diff
@@ -0,0 +1,50 @@
+#ifndef HTTPSESSION_HPP_
+#define HTTPSESSION_HPP_
+
+#include "tcpip_adapter.h"
+
+#include "lwip/err.h"
+#include "lwip/sockets.h"
+#include "lwip/sys.h"
+
+#include "request.hpp"
+#include "response.hpp"
+
+#ifndef HTTD_SERVER_MAX_REQUEST_LENGTH
+#define HTTD_SERVER_MAX_REQUEST_LENGTH 1024
+#endif
+
+namespace httpd
+{
+
+	class Session
```

Comment:
Different indentation than other files, please use 2 spaces for each indent level everywhere.